src/KafkaFlow.Serializer.SchemaRegistry.ConfluentProtobuf/ProducerConfigurationBuilderExtensions.cs

```diff
@@ -1,9 +1,8 @@
-﻿namespace KafkaFlow
+﻿namespace KafkaFlow.Serializer.SchemaRegistry
```

Comment:
please undo this change. The configuration builders should be in KafkaFlow root namespace to make it easier for the client application to use